package.json

```diff
@@ -61,14 +61,12 @@
     "@mapbox/sexagesimal": "1.2.0",
     "@mapbox/vector-tile": "^2.0.3",
     "@openstreetmap/id-tagging-schema": "~6.6.0",
-    "@pixi/display": "7.4.0",
-    "@pixi/events": "7.4.0",
     "@rapid-sdk/math": "~1.0.0-pre.3",
     "@rapid-sdk/util": "~1.0.0-pre.3",
     "@rapideditor/country-coder": "~5.3.0",
     "@rapideditor/location-conflation": "~1.4.0",
-    "@rapideditor/pixi-dashed-line": "7.4.0",
-    "@rapideditor/pixi-texture-allocator": "7.4.0",
+    "@rapideditor/pixi-dashed-line": "file:../pixi-dashed-line",
```

Comment:
I'm going to make releases of these plugins today, then we can remove the `file:..` reference and the PR deploys should start working again.